calibration_date: '2025-02-25 13:16:30'
camera_matrix:
  columns: 3
  data:
  - - 662.1611644155646
    - 0.0
    - 303.6525435002138
  - - 0.0
    - 660.833231130345
    - 240.39227664366038
  - - 0.0
    - 0.0
    - 1.0
  rows: 3
chessboard:
  pattern_size:
    columns: 9
    rows: 6
  square_size_meters: 0.08
distortion_coefficients:
  columns: 5
  data:
  - 0.09430925426104649
  - -0.7632563367834484
  - 0.0010394412525122079
  - -0.001029622027735045
  - 1.6213114654266791
  rows: 1
image_size:
  height: 480
  width: 640
rms_reprojection_error: 0.44620571784124247
